app/views/users/index.html.erb

```diff
@@ -1,2 +1,13 @@
-<h1>Users #index page</h1>
-<h2>This is the Users index page</h2>
\ No newline at end of file
+<div class="main-container">
+<% @users.each do |user| %>
+  <div class="users-wrapper flex">
+    <div class="img-cont">
+      <%= image_tag user.photo, :size => "50x50", alt: "user photo" %>
+    </div>
+    <div class="user-details">
+      <h2><%= user.name %></h2>
```

Comment:
- [ ] The names of the users on the user index page should be a link that takes me to the user's show page. Kindly implement that.